mlflow/tracing/types/wrapper.py

```diff
@@ -75,20 +76,25 @@ def inputs(self) -> Dict[str, Any]:
     def outputs(self) -> Dict[str, Any]:
         return self._outputs
 
-    def set_inputs(self, inputs: Dict[str, Any]):
+    def set_inputs(self, inputs: Any):
```

Comment:
@daniellok-db I just removed the `Dict` restriction here cuz it is easier for users to do `span.set_outputs(x)` than `span.set_outputs({"output": x})`. I wish this doesn't complicate the frondend handling too much, but in case if it does, we can wrap the scalar value in dictionary before calling `log_trace()`.

Reply:
Discussed offline: It's ok to accept arbitral object as long as it is serializable.